composer.json

```
@@ -40,7 +38,10 @@
         }
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "phpstan/extension-installer": false
```

If we use the extension installer this should be `yes`:

```suggestion
            "phpstan/extension-installer": true
```